modules/oadp-installing-dpa-1-2-and-earlier.adoc

```diff
@@ -394,13 +394,18 @@ oc get dpa dpa-sample -n openshift-adp -o jsonpath='{.status}'
 
 . Verify the `type` is set to `Reconciled`.
 
-. Verify the `BackupStorageLocation` by running the following command:
+. Verify the backup storage location and confirm that the `PHASE` is `Available` by running the following command:
 +
 [source,terminal]
 ----
-oc get backupStorageLocation -n openshift-adp
+$ oc get backupStorageLocation -n openshift-adp
+----
+.Example output
+[source,yaml]
```

Comment:
It says [source,yaml] but the output is not in yaml format. Should [source,yaml] be [source,terminal]?